flexget/utils/imdb.py

```diff
@@ -62,7 +62,7 @@ def __init__(self):
         self.min_diff = 0.01
         self.debug = False
 
-        self.max_results = 10
+        self.max_results = 50
```

Comment:
i dunno about making it 50, it'll make the lookup way longer and heavier. maybe a lower default?
bonus points for makings this configurable :stuck_out_tongue: